index.html

```diff
@@ -73,22 +73,22 @@ <h4>MORE FROM YOUTUBE</h4>
     </div>
   </aside>
   <div class="categories">
-    <section class="category-section">
-      <button class="category active">All</button>
-      <button class="category">Category 1</button>
-      <button class="category">Category 2</button>
-      <button class="category">Category 3</button>
-      <button class="category">Category 4</button>
-      <button class="category">Category 5</button>
-      <button class="category">Category 6</button>
-      <button class="category">Category 7</button>
-      <button class="category">Category 8</button>
-      <button class="category">Category 9</button>
-    </section>
+    <div class="category-section">
```

Comment:
Please change `<div>` to `<section>`, thereby implementing semantic HTML.
* Semantic HTML refers to syntax that makes the HTML more comprehensible by better defining the different sections and layout of web pages. It makes web pages more informative and adaptable, allowing browsers and search engines to better interpret content.